src/mosquito/metadata.cr

```diff
@@ -0,0 +1,38 @@
+module Mosquito
+  # Provides a real-time metadata store. Data is not cached, which allows
+  # multiple workers to operate on the same structures in real time.
+  #
+  # Each read or write incurs a round trip to the backend.
+  class Metadata
+    property root_key : String
+    getter readonly : Bool
```

Comment:
```suggestion
    getter? readonly : Bool
```

Doing this you wouldn't need the method definition on 36. Then all of the `if readonly` becomes `if readonly?`

Reply:
thank you, I wanted to find out what the incantation for that was but it seemed like too much work 🍰